src/run.js

```diff
@@ -0,0 +1,5 @@
+export default run => {
+  // setTimeout is fixing this bug:
+  // https://github.com/meteor/react-packages/issues/99
+  setTimeout(run);
```

Comment:
This is basically just an alias for `setTimeout` so it doesn't really need to exist

or if it stays it could be written like
```javascript
export default setTimeout;
```